### ISSUE TITLE
RadioTap - Invalid length

### ISSUE THREAD

**Issue by llamasoft** [secdev/scapy#1457]
String encoded `RadioTap`s appear to have a trailing `\x00` that shouldn't be there:

```
>>> scapy._version()
'2.4.0.dev232'

>>> rt = scapy.layers.dot11.RadioTap()
>>> rt
<RadioTap  |>

>>> str(rt)
'\x00\x00\x08\x00\x00\x00\x00\x00\x00'

>>> len(str(rt))
9
```

The string dump shows that it has a defined length of `\x08\x00` (little-endian unsigned short 8), but the output representation has length 9.

**Comment by gpotter2** [comment]
Hi!
This is a well-known bug, fixed in https://github.com/secdev/scapy/pull/1440 (waiting to be merged)

Could you check that the patch works ?

Thanks !

**Comment by llamasoft** [comment]
Yep, it seems to fix the RadioTap length issue for me!

**Comment by p-l-** [comment]
Fixed in #1440.
